Signal/src/ViewControllers/OWSTableViewController.h

```diff
@@ -72,6 +72,10 @@ typedef UITableViewCell *_Nonnull (^OWSTableCustomCellBlock)();
 
 + (OWSTableItem *)actionItemWithText:(NSString *)text actionBlock:(nullable OWSTableActionBlock)actionBlock;
 
++ (OWSTableItem *)softCenterLabelItemWithText:(NSString *)text;
```

Comment:
Perhaps not a great name, but these "soft, centered label" items are a common part of our design language in our tables.